zim/parser.py

```diff
@@ -334,7 +334,7 @@ def _process(self, builder, text):
 		# default action for matched text
 		if self.descent:
 			builder.start(self.tag)
-			self.descent(builder, *text)
+			self.descent(builder, text)
```

Comment:
I haven't found a use case of the `descent` feature in the present code. I think the "*" was here by a mistake.